pkgs/servers/trezord/default.nix

```diff
@@ -1,65 +1,31 @@
-{ stdenv, fetchgit, fetchFromGitHub, curl, cmake, boost, gcc, protobuf, pkgconfig, jsoncpp
-, libusb1, libmicrohttpd
-}:
+{ stdenv, buildGoPackage, fetchFromGitHub, clang }:
 
-let
-  version = "1.2.1";
-in
+buildGoPackage rec {
+  name = "trezord-go-${version}";
+  version = "2.0.12";
 
-stdenv.mkDerivation rec {
-  name = "trezord-${version}";
+  buildInputs = [clang];
```

Comment:
Rather than using `clang` to work around the issue you can use `hardeningDisable = [ "fortify" ];`.

See https://github.com/NixOS/nixpkgs/blob/b81ca435e6367ddbca29846bfb6ae3b538e67a6c/pkgs/applications/altcoins/go-ethereum.nix for an example

Reply:
Done.